tests/cpp/test_multidevice_transformer.cpp

```diff
@@ -401,11 +402,21 @@ TEST_P(DistributedTransformerTest, MLP_Layer) {
       reference_outs[2],
       reference_outs[3]};
 
-  MultiDeviceExecutor runtime(
-      std::move(fusion), *communicator_, executor_params_);
+  FusionExecutorCache fec(std::move(fusion));
   at::manual_seed(getATenRandomSeed());
-  auto outputs = runtime.runWithInput(inputs);
-  validate(expected_outputs, outputs);
+  auto outputs = fec.runFusionWithInputs(inputs);
+  ASSERT_EQ(outputs.size(), expected_outputs.size());
```

Comment:
Why not just use the existing validate and change the tolerance?

Reply:
I'll try to do that later. This PR only fixes the ref of MLP, and `validate` is used by other tests as well.

Reply:
you could parameterize it with the lower atol. I imagine if we want tighter bounds for all the tests each one will have it's own value that is hard to predict without manually trying